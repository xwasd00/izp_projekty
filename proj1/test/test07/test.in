radek bez cisel
rodne cislo: 113112/3211
