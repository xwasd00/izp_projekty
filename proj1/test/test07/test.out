radek bez cisel
rodne cislo: Y1Y11Y/3211
